core/src/main/java/bio/terra/pearl/core/service/datarepo/DatasetService.java

```diff
@@ -17,14 +17,20 @@
 @Service
 public class DatasetService extends CrudService<Dataset, DatasetDao> {
 
-    private DataRepoClient dataRepoClient;
+    DataRepoClient dataRepoClient;
```

Comment:
Accidental change? I don't see any usage of this that requires a change in visibility.

Reply:
Sorry, not sure why that changed. Fixed in 32e90a08